content/en/docs/contribute/generate-ref-docs/metrics-reference.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Generating reference documentation for metrics
+content_type: task
+weight: 100
+---
+
+<!-- overview -->
+
+This page demonstrates the generation of metrics reference documentation
+
+## {{% heading "prerequisites" %}}
+
+{{< include "prerequisites-ref-docs.md" >}}
+
+<!-- steps -->
+
+## Cloning the Kubernetes repository
+
+The metric generation happens in the Kubernetes repository. 
+To clone the repository, Change directories to where you want the clone to exist.
+
+Then, execute the following command:
+
+```shell
+git clone https://www.github.com/kubernetes/kubernetes 
+```
+
+This creates a `kubernetes` folder in your current working directory.
+
+## Generate the metrics
+
+Inside the cloned repository, You will find a `test/instrumentation/documentation` directory.

```

Comment:
Inside the cloned Kubernetes repository, locate the `test/instrumentation/documentation` directory. The metrics documentation is generated in this directory.